src/main/java/in/projecteka/consentmanager/ConsentManagerApplication.java

```diff
@@ -37,6 +38,7 @@
                                 ListenerProperties.class,
                                 GatewayServiceProperties.class,
                                 RabbitmqOptions.class,
+                                NHSProperties.class,
```

Comment:
it should be NHS or NHA?

Reply:
I'm not sure about that, it just contains hiuId for now.